test_runner/performance/test_perf_pgbench.py

```diff
@@ -79,7 +79,7 @@ def run_test_pgbench(env: PgCompare, scale: int, duration: int):
     # Run simple-update workload
     run_pgbench(env,
                 "simple-update",
-                ['pgbench', '-n', '-c4', f'-T{duration}', '-P2', '-Mprepared', env.pg.connstr()])
+                ['pgbench', '-N', '-c4', f'-T{duration}', '-P2', '-Mprepared', env.pg.connstr()])
```

Comment:
`-n` option is `--no-vacuum`. To run `simple-update` workflow, should specify `-N` option instead.

Reply:
Good catch!